src/modules/xkeyboard.cpp

```diff
@@ -141,9 +140,9 @@ namespace modules {
         }
 
         auto indicator_on = m_keyboard->on(it);
-        auto &indicator_labels = indicator_on ? m_indicator_on_labels : m_indicator_off_labels;
-        auto &indicator_icons = indicator_on ? m_indicator_icons_on : m_indicator_icons_off;
-        auto &indicator_state = indicator_on ? m_indicator_state_on : m_indicator_state_off;
+        auto& indicator_labels = indicator_on ? m_indicator_on_labels : m_indicator_off_labels;
+        auto& indicator_icons = indicator_on ? m_indicator_icons_on : m_indicator_icons_off;
+        auto& indicator_state = indicator_on ? m_indicator_state_on : m_indicator_state_off;
```

Comment:
This seems strange. I do not have a lot of experience in C++, it this syntactically correct? (yes, I am looking to learn something here, I honestly do not know)

Reply:
For the changes to these lines? It was the formatter you run when submitting a PR. The lines themselves use the ternary operator, which is a shortened if-statement. It reads `if ? then : else`.

So these lines ask whether the indicator is on; if it is, they display the "on" icons and labels, otherwise they display the "off" icons and labels.

For simple conditional checks, they seem to be preferred because its very concise. It's pretty common in c-style languages. I'm a beginner with cpp as well, so I'm not sure what the `auto &` is. My best guess is that it becomes the type of the object first assigned to it, and it gets the address of that object? Don't quote me on that.

Reply:
The ternary was the part I understood. Maybe I should have mentioned that :-) 

thanks for researching the `auto&`, I did a short search for this as well earlier (before reading your comment). The type-inference I have seen before, just the "inference and store as reference"-syntax was new. 

Thank you for teaching me (even if by accident and automatic code-formatting) :-)